### PR TITLE
WIP async stack traces ...

### PR DESCRIPTION
**Pull request by ry**
cc @nayeem-rahman 

```
> ./tools/build.py libdeno_test && ./target/debug/libdeno_test --gtest_filter=LibDenoTest.AsyncException
ninja: Entering directory `/Users/rld/src/deno/target/debug'
[2/2] LINK ./libdeno_test
Note: Google Test filter = LibDenoTest.AsyncException
[==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from LibDenoTest
[ RUN      ] LibDenoTest.AsyncException
Error
    at /Users/rld/src/deno/core/libdeno/libdeno_test.js:260:21
    at async /Users/rld/src/deno/core/libdeno/libdeno_test.js:259:5
deno_last_exception (null)
deno_last_exception {"message":"Uncaught Error","sourceLine":"","lineNumber":0,"startPosition":-1,"endPosition":-1,"errorLevel":8,"startColumn":-1,"endColumn":-1,"isSharedCrossOrigin":true,"isOpaque":false,"frames":[{"line":260,"column":21,"scriptName":"/Users/rld/src/deno/core/libdeno/libdeno_test.js","isEval":false,"isConstructor":false,"isWasm":false}]}
[       OK ] LibDenoTest.AsyncException (80 ms)
[----------] 1 test from LibDenoTest (80 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test suite ran. (80 ms total)
[  PASSED  ] 1 test.
```

Other places to look
* `isolate_->SetPromiseRejectCallback()` https://github.com/denoland/deno/blob/deec1b9b97011d1a0ef7312c0a3efde21186b82d/core/libdeno/binding.cc#L562

### CODE REVIEW

File: core/libdeno/api.cc [hunk 1]
```diff
@@ -120,8 +120,8 @@ void deno_init() {
     // remove this to make it work asynchronously too. But that requires getting
     // PumpMessageLoop and RunMicrotasks setup correctly.
     // See https://github.com/denoland/deno/issues/2544
-    const char* argv[2] = {"", "--no-wasm-async-compilation"};
-    int argc = 2;
+    const char* argv[3] = {"", "--no-wasm-async-compilation", "--async-stack-traces"};
```
Comment (nayeem-rahman) [inline core/libdeno/api.cc]: I think `--async-stack-traces` is on by default. All it's specified to do is enrich `error.stack`: https://v8.dev/docs/stack-trace-api#async-stack-traces

Comment (ry) [reply core/libdeno/api.cc]: It doesn’t seem to have an effect - I was just trying it

Comment (kitsonk) [reply core/libdeno/api.cc]: I think they have to be collected via other APIs 😢 see: https://github.com/nodejs/node/issues/11865#issuecomment-288450455

**Comment by ry** [comment]
stale

**Pull request closed without merging**